app/src/main/res/layout/fragment_image_filter.xml

```diff
@@ -0,0 +1,125 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent" android:layout_height="match_parent"
+    >
+
+    <TextView
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="@string/img_size"
+        android:id="@+id/tv_img_size"
+        android:layout_alignBottom="@+id/sp_imgsize"
+        android:layout_alignParentLeft="true"
+        android:layout_alignParentStart="true"
+        android:layout_alignParentTop="true"
+        android:gravity="center_vertical"
+        android:textSize="18sp"
+        android:paddingBottom="10dp"
+        android:layout_toLeftOf="@+id/sp_imgsize"
+        android:layout_toStartOf="@+id/sp_imgsize" />
+
+    <Spinner
+        android:layout_width="fill_parent"
+        android:layout_height="wrap_content"
+        android:id="@+id/sp_imgsize"
+        android:layout_alignParentTop="true"
+        android:layout_toEndOf="@+id/tv_imgcolor"
+        android:spinnerMode="dropdown"
+        android:textSize="18sp"
+        android:minHeight="45dp"
+        android:paddingBottom="10dp"
+        android:layout_toRightOf="@+id/tv_imgcolor" />
```

Comment:
@edwinperaza you can initialize the spinners with static options via the `android:entries=` property. Fro example, in this particular spinner you can do `android:entries=@array/sp_img_sz`. Similar treatment can be done to the other spinners.